### PR TITLE
Remove lodash-fp and full lodash imports from personalization apps

### PR DESCRIPTION
**Pull request by theodur**
## Description
This PR removes full lodash imports from personalization apps.

cc @bradl3yC 

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by bradl3yC — APPROVED** [review]

**Review by ncksllvn — APPROVED** [review]

File: src/applications/personalization/dashboard/reducers/folders.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import _ from 'lodash';
-import set from 'lodash/fp/set';
+import kebabCase from 'lodash/kebabCase';
+import set from 'platform/utilities/data/set';
```
Comment (erikphansen) [inline src/applications/personalization/dashboard/reducers/folders.js:2]: the usage of our platform `set` is the same as lodashfp's `set`?

File: src/applications/personalization/dashboard/reducers/prescriptions.js [hunk 2]
```diff
@@ -69,7 +68,7 @@ export default function prescriptions(state = initialState, action) {
         };
       }
 
-      return assign(state, newState);
+      return Object.assign({}, state, newState);
```
Comment (erikphansen) [inline src/applications/personalization/dashboard/reducers/prescriptions.js:71]: might as well just do `{...state, ...newState}`?

File: src/applications/personalization/dashboard/reducers/folders.js (hunk 1 above)
Comment (ncksllvn) [reply src/applications/personalization/dashboard/reducers/folders.js:2]: I think the function signature is different

Comment (erikphansen) [reply src/applications/personalization/dashboard/reducers/folders.js:2]: Hmm there are a couple places in this PR where the swap is made but no change to usage. Can you double check that @theodur ?

Comment (theodur) [reply src/applications/personalization/dashboard/reducers/folders.js:2]: The function signature for lodash's set is different, but the signature for [lodash/fp's set](https://gist.github.com/jfmengels/6b973b69c491375117dc#_setpath-value-object) is the same as the one we use in platform.

File: src/applications/personalization/dashboard/reducers/prescriptions.js (hunk 2 above)
Comment (theodur) [reply src/applications/personalization/dashboard/reducers/prescriptions.js:71]: Yeah I agree, this was ported over from a larger PR that got split.

File: src/applications/personalization/dashboard/reducers/folders.js (hunk 1 above)
Comment (erikphansen) [reply src/applications/personalization/dashboard/reducers/folders.js:2]: Thanks for confirming! I would hope unit tests would have failed if this wasn’t the case, but still.

**Pull request closed without merging**